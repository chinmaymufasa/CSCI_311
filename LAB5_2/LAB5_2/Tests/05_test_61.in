2
165
5
2
61
4
2
163
4
2
131
5
2
117
6
2
116
4
2
178
5
2
17
5
9
3
590
5
9
2
176
4
9
3
178
4
9
2
191
5
2
40
4
2
92
5
2
49
4
2
113
4
2
137
5
2
127
6
9
3
799
6
9
2
150
6
2
18
5
2
195
6
9
3
165
6
9
2
108
4
-1
