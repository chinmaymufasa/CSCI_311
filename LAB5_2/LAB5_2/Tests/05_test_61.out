165 
165 61 
163 61 165 
61 131 163 165 
61 131 117 165 163 
117 61 116 163 131 165 
61 116 117 131 163 165 178 
17 61 116 117 131 163 165 178 
8
17 61 116 117 131 163 165 178 
8
117 61 17 116 163 131 176 165 178 
9
117 61 17 116 163 131 176 165 
8
17 61 116 117 131 163 165 176 191 
117 61 17 40 116 163 131 176 165 191 
17 40 61 92 116 117 131 163 165 176 191 
117 61 40 17 49 116 92 163 131 176 165 191 
117 61 40 17 49 113 92 116 163 131 176 165 191 
17 40 49 61 92 113 116 117 131 137 163 165 176 191 
17 49 40 92 116 113 61 127 137 131 165 191 176 163 117 
15
17 49 40 92 116 113 61 127 137 131 165 191 176 163 117 
15
17 49 40 92 116 113 61 127 150 137 131 165 191 176 163 117 
17 18 40 49 61 92 113 116 117 127 131 137 150 163 165 176 191 
18 17 49 40 92 116 113 61 127 150 137 131 165 195 191 176 163 117 
18
18 17 49 40 92 116 113 61 127 150 137 131 176 195 191 163 117 
17
117 61 40 17 18 49 113 92 108 116 163 131 127 137 150 191 176 195 
